tests/tasks/create_macvlan_interface_no_autoconnect.yml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- include_role:
+    name: linux-system-roles.network
+  vars:
+    network_connections:
+      - name: eth0-profile 
+        autoconnect: false
+        type: ethernet 
+        interface_name: eth0
+        ip:
+          address:
+            - 192.168.0.1/24
+      - name: "{{ interface }}"
+        type: macvlan
+        parent: eth0-profile
+        macvlan:
+          mode: bridge
+          promiscuous: yes
+          tap: no
+        ip:
+          address:
+            - 192.168.1.1/24
+- debug:
+    var: __network_connections_result
+...
```

Comment:
you don't need the standard yaml trailing line with `...` or the extra blank line at the end

Reply:
Got it, already delete it~